app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self: None, comfort_class: None,
+                 clean_mark: None,
+                 brand: None) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self: None, distance_from_city_center: None,
```

Comment:
Reformat your code so that all arguments in the `__init__` including `self` would start from a new line.